examples/policies/policy-cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
```

Comment:
what will this policy do if the cluster-monitoring-config configmap already existed

Reply:
@marcolan018 seems it does nothing - works well. I just tested it. Check the jb-4 Hub server that I slacked you this am.